extensions/cornerstone-dicom-sr/src/commandsModule.ts

```diff
@@ -42,8 +43,57 @@ const _generateReport = (measurementData, additionalFindingTypes, options = {})
 
 const commandsModule = (props: withAppTypes) => {
   const { servicesManager } = props;
-  const { customizationService } = servicesManager.services;
+  const { customizationService, measurementService, viewportGridService, uiDialogService } =
+    servicesManager.services;
   const actions = {
+    jumpToMeasurement: ({ uid }) => {
+      measurementService.jumpToMeasurement(viewportGridService.getActiveViewportId(), uid);
+    },
+
+    removeMeasurement: ({ uid }) => {
+      measurementService.remove(uid);
+    },
+
+    renameMeasurement: ({ uid }) => {
+      const labelConfig = customizationService.get('measurementLabels');
+      const measurement = measurementService.getMeasurement(uid);
+      showLabelAnnotationPopup(measurement, uiDialogService, labelConfig).then(val => {
+        measurementService.update(
+          uid,
+          {
+            ...val,
+          },
+          true
+        );
+      });
+    },
+
+    changeColorMeasurement: ({ uid }) => {
+      const { color } = measurementService.getMeasurement(uid);
+      const rgbaColor = {
+        r: color[0],
+        g: color[1],
+        b: color[2],
+        a: color[3] / 255.0,
+      };
+      colorPickerDialog(uiDialogService, rgbaColor, (newRgbaColor, actionId) => {
```

Comment:
This is misleading because we don't support re-color. We should consider throwing an error to indicate it's not supported yet. I think this was my mistake.